BUILD

```diff
@@ -1 +1,15 @@
 # This file exists so that the intellij Bazel plugin has something to bootstrap the project from
+
+load("@com_github_bazelbuild_buildtools//buildifier:def.bzl", "buildifier")
+
+buildifier(
```

Comment:
If we want to add this, let's do it in a separate MR

Reply:
Ok!